pom.xml

```diff
@@ -23,33 +23,21 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-data-jpa</artifactId>
         </dependency>
+        <dependency>
```

Comment:
Make sure to merge the other branch first and pull from it (or rather, do merge develop into this branch) so that you have all the changes on that branch before merging.

Reply:
This has been achieved, please cross-check on you end